jplag/src/main/java/de/jplag/SubmissionSetBuilder.java

```diff
@@ -248,23 +287,26 @@ private Submission processSubmission(String submissionName, File submissionFile)
         }
 
         submissionFile = makeCanonical(submissionFile, it -> new SubmissionException("Cannot create submission: " + submissionName, it));
-        return new Submission(submissionName, submissionFile, parseFilesRecursively(submissionFile), language, errorCollector);
+        return new Submission(submissionName, submissionFile, submissionsNeedChecking, parseFilesRecursively(submissionFile), language,
```

Comment:
Again, something like `isOld `would probably be better, as that is the reason a submission might need no checking.